test-src/manageworkertype_test.js

```diff
@@ -57,3 +59,237 @@ describe('provisioner worker type api', () => {
     }
   });
 });
+
+describe('worker-type API methods', () => {
+  let WorkerType, WorkerState, wt, testWorkerType, testWorkerState;
+
+  let cleanTables = async () => {
+    // remove all rows from either table
+    await WorkerType.scan({}, {
+      handler: async (item) => { await item.remove();  },
+    });
+    await WorkerState.scan({}, {
+      handler: async (item) => { await item.remove();  },
+    });
+  };
+
+  before(async () => {
+    WorkerState = workerState.setup({
+      table: helper.cfg.get('provisioner:workerStateTableName'),
+      credentials: helper.cfg.get('azure'),
+    });
+
+    let keyPrefix = helper.cfg.get('provisioner:awsKeyPrefix');
+    let pubKey = helper.cfg.get('provisioner:awsInstancePubkey');
+    let provisionerId = helper.cfg.get('provisioner:id');
+    let provisionerBaseUrl = helper.cfg.get('server:publicUrl') + '/v1';
+    WorkerType = workerType.setup({
+      table: helper.cfg.get('provisioner:workerTypeTableName'),
+      credentials: helper.cfg.get('azure'),
+      context: {keyPrefix, provisionerId, provisionerBaseUrl, pubKey},
+    });
+
+    await WorkerState.ensureTable();
+    await WorkerType.ensureTable();
+    await cleanTables();
+
+    wt = slugid.nice();
+    testWorkerType = {
```

Comment:
this still isn't using the mock-workers.js file.  Using that file means that when we do schema changes, we only need to update that single file to catch almost all instances of the update to the schema.